sample/model/sample_special_joint_robot.wrl

```diff
@@ -0,0 +1,1023 @@
+#VRML V2.0 utf8
+#--------------------------------------------------------------
+# OpenHRP Sample Special Joint Robot Model
```

Comment:
OpenHRP Sample Special Joint (Mimic Joint, 6DOF Joint for fly wheel .... ) Robot Model